### ISSUE TITLE
Phonepe - Compliance Check

### ISSUE THREAD

**Issue by BLR-0118** [ONDC-Official/Pre-production#167]
1. /search - payment type "ON-ORDER" not applicable here;
2. /select - payment type "ON-ORDER" not applicable here, buyer app finder fee only sent in /search;
3. /init - there's no quote in the API contract;
4. /confirm - quote.price.value is 146 whereas the same in /on_select is 146.16;
5. /confirm - no payment.params details such as amount collected, transaction_id;
6. /status - timestamp for this request is 14th Sep using an order id created in /confirm on 15th Sep which returns /on_status showing items approved for return. How?
7. /track - same issue as /status
8. No /update, /support API?

**Comment by BLR-0118** [comment]
@batramanuj

**Comment by batramanuj** [comment]
@BLR-0118 : As discussed

1. /search - payment type "ON-ORDER" not applicable here;
- We were sending it as innobits was expecting it.Will remove this and share the updated logs once change is done from innobits side.
2. /select - payment type "ON-ORDER" not applicable here, buyer app finder fee only sent in /search;
- We were sending it as innobits was expecting it.Will remove this and share the updated logs once change is done from innobits side.
3. /init - there's no quote in the API contract;
- This is an extra info which was coming from our side which will be fixed in the updated logs.
4. /confirm - quote.price.value is 146 whereas the same in /on_select is 146.16;
- We had not handled decimal values as we were waiting for the new price contract to be published.As discussed with you for CUG we will be assuming price coming from NPs to be rounded off to 2 decimal points as the price contract change is not in scope of CUG.This change needs to be communicated to all NPs.
5./confirm - no payment.params details such as amount collected, transaction_id;
- This will be fixed in the updated logs
6./status - timestamp for this request is 14th Sep using an order id created in /confirm on 15th Sep which returns /on_status showing items approved for return. How?
- This was an issue on our end and will be fixed in the updated logs
7./track 
- We don't want to open external links in our app and hence will not be calling the track api.We will rely on order and fulfilment states to communicate the current state of the order to the user
8. No /update, /support API?
- We don't have a use case for calling update api as there is no editing happening from the user post order confirmation except for cancellation ,which will be done through cancel api.
- For user support on our app, each user query will generate a ticket in our CS ticketing system and user query resolution will be taken care of by our CS team .We don't want to give inconsistent experience to our users by making them directly talk to the store using the data coming through support api for any issue resolution.As far as user enquires are concerned w.r.t. any store, we will wait for store contact details to be available as part of the search call rather than using the support api for this.

**Comment by batramanuj** [comment]
@abaweja

**Comment by BLR-0118** [comment]
1. diff in quote between /on_select & /on_init - delivery charge is added in /on_init but not in /on_select. Won't this cause a problem with the buyer experience since the quote they get after putting items in their cart is expected to include product + service costs.
2. Where's the log for /update API?

**Comment by BLR-0118** [comment]
@batramanuj @abaweja
